modules/oauth-configuring-internal-oauth.adoc

```diff
@@ -33,8 +33,8 @@ spec:
     accessTokenMaxAgeSeconds: 172800 <1>
 ----
 <1> Set `accessTokenMaxAgeSeconds` to control the lifetime of access tokens.
-The default lifetime is 24 hours, or 86400 seconds. This attribute cannot
-be negative.
+The default lifetime is 24 hours, or 86400 seconds. This attribute must
+be positive.
```

Comment:
:+1: maybe let's add the special case as it is a bit unexpected. something along the lines:
```
If set to zero, the default lifetime will be used.
```

Reply:
Ok, added.